### PR TITLE
moved cleanup volsync to a new func

### PR DESCRIPTION
**Pull request by rakeshgm**
moved cleanup volsync to a new function

Signed-off-by: rakeshgm <rakeshgm@redhat.com>

### CODE REVIEW

**Comment by rakeshgm** [comment]
the unit tests are failing here. but the tests are passing in my fork. I believe the change in this PR does not impact any tests directly. here is the test passing in my fork: [tests](https://github.com/rakeshgm/ramen/actions/runs/3694726997). can the tests be  retriggered

**Comment by rakeshgm** [comment]
> the unit tests are failing here. but the tests are passing in my fork. I believe the change in this PR does not impact any tests directly. here is the test passing in my fork: [tests](https://github.com/rakeshgm/ramen/actions/runs/3694726997). can the tests be retriggered?

I believe the tests are passing now.

File: controllers/drplacementcontrol.go [hunk 1]
```diff
@@ -1344,41 +1343,7 @@ func (d *DRPCInstance) EnsureCleanup(clusterToSkip string) error {
...
-
-		if !peersReady {
-			return fmt.Errorf("still waiting for peer to be ready")
-		}
-
-		d.setDRPCCondition(&d.instance.Status.Conditions, rmn.ConditionPeerReady, d.instance.Generation,
-			metav1.ConditionTrue, rmn.ReasonSuccess, "Ready")
-
-		return nil
+		return d.setDRPCConditionForVolsync(clusterToSkip)
```
Comment (BenamarMk) [inline controllers/drplacementcontrol.go:1346]: Maybe it should be renamed to `cleanupForVolSync` or `cleanupSecondariesForVolSync`

Comment (rakeshgm) [reply controllers/drplacementcontrol.go]: renamed to`cleanupForVolSync`

**Review by BenamarMk — APPROVED** [review]

**Pull request merged**